src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -1366,13 +1368,31 @@ TEST_CASE_VERSIONS("refund test with closeLedger", "[tx][soroban][feebump]")
         // resource fee, so the total cost would be greater than
         // DEFAULT_TEST_RESOURCE_FEE without the refund.
         REQUIRE(txFeeWithRefund < DEFAULT_TEST_RESOURCE_FEE);
+
```

Comment:
I would add some additional tests. A couple off the top of my head- 
1. Classic transaction.
2. Classic transaction that makes a payment within the transaction.
3. Classic fee bump.